lib/database/achievement.php

```diff
@@ -79,7 +79,7 @@ function getAchievementsList(
     }
 
     $query = "SELECT
-                    ach.ID, ach.Title AS AchievementTitle, ach.Description, ach.Points, ach.TrueRatio, ach.Author, ach.DateCreated, ach.DateModified, ach.BadgeName, ach.GameID,
+                    ach.ID, ach.Title AS Title, ach.Title AS AchievementTitle, ach.Description, ach.Points, ach.TrueRatio, ach.Author, ach.DateCreated, ach.DateModified, ach.BadgeName, ach.GameID,
```

Comment:
I have a feeling these changes will affect the public API as they mostly just dump the query objects. I suppose duplicated fields won't hurt anything, unless you're concerned about bandwidth.